packages/sol-rayz/src/getParsedAccountByMint.ts

```diff
@@ -0,0 +1,27 @@
+// @ts-nocheck
```

Comment:
That's not good. It ruins the idea of using TS 🙆🏽‍♂️

Reply:
Yeap. But we use it everywhere before. Probably we need to prioritize refactoring it to ts.
<img width="775" alt="Screenshot 2021-11-16 at 15 08 59" src="https://user-images.githubusercontent.com/18005988/141990971-05b5bfe4-639f-42e3-9767-0e5bada23c5d.png">